src/index.ts

```diff
@@ -85,6 +85,19 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     configurations.pollingInterval = MIN_POLLING_INTERVAL
   }
 
+  if (configurations.streamEnabled) {
+    try {
+      const { Platform } = require("react-native");
+      if  (Platform.OS === 'android') {
+        console.warn("SDKCODE:1007 workaround applied - Android React Native detected - streaming will be disabled and polling enabled")
```

Comment:
This is a nitpick, but I'd suggest take out the wording 'workaround applied' and just leave the rest of the warning message.